src/sentry/incidents/subscription_processor.py

```diff
@@ -212,11 +212,20 @@ def get_comparison_aggregation_value(
                     "end": end,
                 },
             )
-            time_col = ENTITY_TIME_COLUMNS[get_entity_key_from_query_builder(query_builder)]
+            entity_key = get_entity_key_from_query_builder(query_builder)
+            time_col = ENTITY_TIME_COLUMNS[entity_key]
             query_builder.add_conditions(
                 [
-                    Condition(Column(time_col), Op.GTE, start),
-                    Condition(Column(time_col), Op.LT, end),
+                    Condition(
+                        Column(time_col, entity=Entity(entity_key.value, alias="entity")),
```

Comment:
Is the alias required here? I don't think you need to pass it, it's used later on when parsing the results (I think?)

Reply:
I asked @kylemumma that also. 🤔

Reply:
Resolving because it is required